src/Generator/Generators/CLI/CLITypeReferences.cs

```diff
@@ -142,7 +142,7 @@ private void GenerateInclude(ASTRecord<Declaration> record)
         private string GetIncludePath(TranslationUnit translationUnit)
         {
             if (!DriverOptions.UseHeaderDirectories)
-                return translationUnit.FileName;
+                return $"{DriverOptions.GenerateName(translationUnit)}.h";
```

Comment:
There are cases when headers have an extension other than ".h", even no extension at all. Handle this case in your logic.